package.json

```diff
@@ -61,26 +51,12 @@
          ],
          "aksdevx.submenuDraft": [
             {
-               "submenu": "aksdevx.aksDraftSubMenu",
-               "group": "draftnavigation"
-            }
-         ],
-         "aksdevx.aksDraftSubMenu": [
-            {
-               "command": "aks-draft-extension.runDraftCreate",
+               "command": "aks-draft-extension.runDraftDockerfile",
                "group": "draftnavigation@1"
             },
             {
-               "command": "aks-draft-extension.runDraftSetupGH",
+               "command": "aks-draft-extension.runDraftDeployment",
                "group": "draftnavigation@2"
-            },
```

Comment:
💡 Thank you @qpetraroia for quick conversation, since this is removing whole lot of existing functionality, shall we open a tracking issue against repo and Pin it as a `breaking change` for anyone or 1000 users using this extension in current state. Just an Idea I thought I will FYI.

Reply:
Thanks for the review @Tatsinnit, yes that is something we will be doing! Documentation and a pinned issue will be created/merged before this change goes live